pkg/apps/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -121,20 +121,21 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		return c.updateStatus(config, existingDeployments)
 	}
 
-	latestIsDeployed, latestDeployment := deployutil.LatestDeploymentInfo(config, existingDeployments)
+	latestExists, latestDeployment := deployutil.LatestDeploymentInfo(config, existingDeployments)
```

Comment:
this is more "latestControllerCreated" :)

Reply:
I think I made it to be consistent with some other occurrences in our code
yes, (latestRCCreated || latestRCRunning || latestRCDeployed) - I guess exists covers it but I don't really have a preference except we stop calling it latestDeployed which I find confusing because it implies it is in terminal state